code/renderers/svelte/src/decorators.ts

```diff
@@ -1,89 +1,94 @@
 import type { DecoratorFunction, StoryContext, LegacyStoryFn } from '@storybook/types';
 import { sanitizeStoryContextUpdate } from '@storybook/preview-api';
-import SlotDecorator from '../templates/SlotDecorator.svelte';
+// ! DO NOT change this SlotDecorator import to a relative path, it will break it.
+// ! A relative import will be compiled at build time, and Svelte will be unable to
+// ! render the component together with the user's Svelte components
+// ! importing from @storybook/svelte will make sure that it is compiled at runtime
+// ! with the same bundle as the user's Svelte components
+// eslint-disable-next-line import/no-extraneous-dependencies
+import SlotDecorator from '@storybook/svelte/templates/SlotDecorator.svelte';
 import type { SvelteRenderer } from './types';
 
-/**
- * Check if an object is a svelte component.
- * @param obj Object
- */
-function isSvelteComponent(obj: any) {
-  return obj.prototype && obj.prototype.$destroy !== undefined;
-}
-
 /**
  * Handle component loaded with esm or cjs.
  * @param obj object
  */
-function unWrap(obj: any) {
-  return obj && obj.default ? obj.default : obj;
+function unWrap<T>(obj: T): T {
+  return (obj as any)?.default || obj;
```

Comment:
@kasperpeulen is there a better way to type this than with `any`?

I want the function to return the same type as it got in, but if I don't assign it to `any` here it will complain that `default` doesn't exist on `T`, which is fair.

Reply:
You could do this:
```ts
function unWrap<T>(obj: { default: T } | T): T {
  return obj && typeof obj === 'object' && 'default' in obj ? obj.default : obj;
}
```